.github/workflows/dep-updates_pr-tracking-branch-to-default.yml

```diff
@@ -2,7 +2,7 @@ name: Create batch dependency update PR
 
 on:
   schedule:
-    - cron: "35 10 * * MON"
+    - cron: "35 10 1 * *"
```

Comment:
If I edit via GitHub (which I did) there's an explainer for the cron that pops up on hover. It appears to recognise the syntax used here as at 10.35AM on the first day of the month.  Also recognised by https://crontab.guru/#35_10_1_*

I think it's all above board(?) because day of week is `*`, although reading the [manual](https://www.unix.com/man-page/POSIX/1posix/crontab/) it looks like specifying that would also be valid (but not what we're aiming for).

> If either the month or day of month is specified as an element or list, but the day of week is an asterisk, the month and day of month fields shall specify the days that match.